features/credentials/views/provider_credentials.erb

```diff
@@ -0,0 +1,69 @@
+<div class="mdl-layout mdl-js-layout mdl-layout--fixed-drawer">
+  <header class="mdl-layout__header">
+    <div class="mdl-layout__header-row">
+      <span class="mdl-layout__title"><%= title %></span>
+    </div>
+  </header>
+
+  <%= erb :"../../partials/navigation" %>
+
+  <main class="mdl-layout__content" style="padding: 40px">
+    <!-- List all existing provider credentials for a user -->
+
+    <h3>Provider Credentials</h3>
+    <table>
+      <thead>
+        <tr>
+          <th>id</th>
```

Comment:
I think for the headers, we could make this a little less ruby and more user friendly, like instead of `api_token` it could just be `API Token`. No big deal either way, just thinking that since we're displaying this, it might be nicer to look at.